### PR TITLE
Iisrum 3257 repeating maintenances dialog changes

### PR DESCRIPTION
**Pull request by kompa3**
The code still has a lot of issues such as
- Misleading / ambiguous component names
- Misleading / ambiguous field and method names
- `IirisMaintenanceModal` has field members on three levels: `state`, `this.scope` and `this`. Probably only `state` would be enough and much more readable & simpler code.
- There are other known issues such as IISCRUM-3219.

I intend to continue refactoring with IISCRUM-3219 ticket but I'd prefer to make PRs in "smaller" chunks so here's the PR for IISCRUM-3257. I have tested it in my local environment and it seems to work. I don't intend to release it to testing though before first tackling the other issues mentioned above.

### CODE REVIEW

File: public/app/features/dashboard/components/DashNav/common_tools.ts [hunk 1]
```diff
@@ -3,8 +3,6 @@
 import _ from 'lodash';
 import moment from 'moment'; // eslint-disable-line no-restricted-imports
 
-const GLOBAL = 'global';
```
Comment (bulkware) [inline public/app/features/dashboard/components/DashNav/common_tools.ts:6]: This was already removed in https://github.com/digiaiiris/grafana/pull/140/files. Just a side note ;)

File: public/app/features/dashboard/components/DashNav/IirisMaintenanceTable.tsx [hunk 2]
```diff
@@ -55,7 +56,7 @@ export function IirisMaintenanceTable(props: Props) {
                         ? 'iiris-colored-row'
                         : '')
                     }
-                    key={`tbody-td-${i}`}
+                    key={`tbody-td-${maintenanceId}-${columnIndex}`}
```
Comment (bulkware) [inline public/app/features/dashboard/components/DashNav/IirisMaintenanceTable.tsx:59]: This is way, way better than the index loop variable.

File: public/app/features/dashboard/components/DashNav/IirisMaintenance.tsx [hunk 3]
```diff
@@ -295,25 +293,6 @@ class IirisMaintenance extends PureComponent<Props, State> {
       });
   };
 
-  /**
```
Comment (bulkware) [inline public/app/features/dashboard/components/DashNav/IirisMaintenance.tsx:298]: Wiki-link is not relevant anymore?

**Review by bulkware — APPROVED** [review]
Good work! Just some side notes and questions.

File: public/app/features/dashboard/components/DashNav/IirisMaintenance.tsx (hunk 3 above)
Comment (kompa3) [reply public/app/features/dashboard/components/DashNav/IirisMaintenance.tsx:298]: This was actually dead code. Wiki link is another component and separate file, IirisServiceInfoWikiButton.tsx.

**Pull request merged**